src/web/components/SeriesSectionLink.tsx

```diff
@@ -71,17 +71,25 @@ const invertedStyle = (pillar: Pillar) => css`
     color: ${neutral[100]};
     background-color: ${pillarPalette[pillar].main};
 
+    /* Handle text wrapping onto a new line */
+    white-space: pre-wrap;
+    box-shadow: -6px 0 0 ${pillarPalette[pillar].main};
+    box-decoration-break: clone;
```

Comment:
Nice solution. IE11 might look odd with no support (though maybe just one big block behind?)

Reply:
Without this clone the second line has no left padding but we've fixed the overlap problem so IE11 experience will still be better than before